appcues/src/main/java/com/appcues/ui/composables/StepComposition.kt

```diff
@@ -22,23 +26,26 @@ internal fun Step.ComposeStep(
     modifier: Modifier = Modifier,
     containerPadding: PaddingValues,
     safeAreaInsets: PaddingValues,
-    parent: BoxScope
+    parent: BoxScope,
+    hasVerticalScroll: Boolean,
 ) {
-    CompositionLocalProvider(
-        LocalAppcuesActions provides actions,
-        LocalExperienceStepFormStateDelegate provides formState
-    ) {
-        // used to get the padding values from step decorating trait and apply to the Column
-        val density = LocalDensity.current
-        val stickyContentPadding = remember(this) { StickyContentPadding(density) }
+    key(id) {
```

Comment:
this line is the fix for the item in the ticket - preventing the compose node re-use between steps that could lead to unexpected text sizing (due to the row using IntrinsicSize.Min for height)